gobcore/events/import_message.py

```diff
@@ -0,0 +1,92 @@
+"""GOB Header
+
+The GOB Header holds the header values of a workflow message.
+
+Todo: Name of the module (message) should correspond to the contents of the file (GOBHeader)
+
+"""
+from gobcore.exceptions import GOBException
+
+
+class MessageMetaData():
+
+    source_id_column = '_source_id'
+
+    def __init__(self, header):
+        self._header = header
+
+    @property
+    def source(self):
+        return self._header['source']
+
+    @property
+    def timestamp(self):
+        return self._header['timestamp']
+
+    @property
+    def id_column(self):
+        return self._header['entity_id']
+
+    @property
+    def entity(self):
+        return self._header['entity']
+
+    @property
+    def version(self):
+        return self._header['version']
+
+    @property
+    def model(self):
+        return self._header['gob_model']
+
+    @property
+    def as_header(self):
+        return {
```

Comment:
Probably return { attr: self[attr] for attr in [...] }

Reply:
I admit I prefer the straightforward readability of the current implementation